src/unicodeMake.js

```diff
@@ -1,70 +0,0 @@
-/* eslint no-console:0 */
-
-// This is an internal Node tool, not part of the KaTeX distribution,
-// whose purpose is to generate unicodeSymbols.js in this directory.
-// In this way, only this tool, and not the distribution/browser,
-// needs String's normalize function.
-const path = require('path');
-const fs = require('fs');
-
-const target = path.join(__dirname, 'unicodeSymbols.js');
-const targetMtime = fs.statSync(target).mtime;
-if (fs.statSync(__filename).mtime <= targetMtime && fs.statSync(
-        path.join(__dirname, 'unicodeAccents.js')).mtime <= targetMtime) {
-    process.exit(0);
-}
-
-require('@babel/register');
-const accents = require('./unicodeAccents').default;
-
-const encode = function(string) {
-    let output = '"';
-    for (let i = 0; i < string.length; i++) {
-        let hex = string.charCodeAt(i).toString(16);
-        while (hex.length < 4) {
-            hex = `0${hex}`;
-        }
-        output += `\\u${hex}`;
-    }
-    output = `${output}"`;
-    return output;
-};
```

Comment:
We don't need this anymore since we aren't generating the file, the preval plugin is, right?

Reply:
@kevinbarabash Yes, escaping characters will be done by `minify`.